pages/common/cbt.md

```diff
@@ -0,0 +1,28 @@
+# cbt
+
+> Utility for reading data from Google Cloud's Bigtable.
+> More information: <https://cloud.google.com/bigtable/docs/cbt-reference>.
+
+- List tables in current project:
+
+`cbt ls`
+
+- Return count of rows in table:
+
+`cbt count {{table_name}}`
```

Comment:
```suggestion
`cbt count "{{table_name}}"`
```

Optional fix: `{{table_name}}` placeholder can be quoted if it can contain some special characters to POSIX-like shell.

Reply:
I double checked the documentation and it doesn't make any notes on character restrictions so might be safer to include these quotes as well